### PR TITLE
Add filter groups to default reports page

### PR DESCRIPTION
**Pull request by crutan**
#### What's this PR do?

Adds a selection widget for filter groups to the default reports page.

#### How should this be manually tested?

Create a filter group that narrows your data set.  Run a report with no filter group selected, observe the number of properties, then select a filter group and update the charts - you should see your property count decrease according to the filter group conditions

#### What are the relevant tickets?

#4809 

#### Screenshots (if appropriate)
image
image

### CODE REVIEW

File: seed/views/v3/organizations.py [hunk 1]
```diff
@@ -850,6 +854,15 @@ def get_raw_report_data(self, organization_id, access_level_instance, cycles, x_
...
 
+        if filter_group_id:
+            filter_group = FilterGroup.objects.get(pk=filter_group_id)
+            if filter_group.query_dict:
+                qd = QueryDict(mutable=True)
+                qd.update(filter_group.query_dict)
+                columns = Column.retrieve_all(org_id=organization_id, inventory_type="property", only_used=False, include_related=False)
+                filters, _annotations, _order_by = build_view_filters_and_sorts(qd, columns, "property")
+                all_property_views = all_property_views.filter(filters)
+
```
Comment (haneslinger) [inline seed/views/v3/organizations.py:865]: Does this handle the filter group's label filtering too? I think you have to do something like in[ this function ](https://github.com/SEED-platform/seed/blob/Fix-element-statistics/seed/models/data_views.py#L31), I I bet you can get it a little cleaner

File: seed/static/seed/js/controllers/inventory_reports_controller.js [hunk 2]
```diff
@@ -56,6 +57,10 @@ angular.module('SEED.controller.inventory_reports', []).controller('inventory_re
     $scope.access_level_instance_id = null;
     $scope.users_access_level_instance_id = user_service.get_access_level_instance().id;
 
+    filter_groups_service.get_filter_groups('Property').then((filter_groups) => {
+      $scope.filter_groups = filter_groups;
+    });
```
Comment (haneslinger) [inline seed/static/seed/js/controllers/inventory_reports_controller.js:62]: This could be done in `seed.js` too but I dont really think it matters 🤷🏻

**Review by haneslinger — CHANGES_REQUESTED** [review]
@crutan, im sure I speak for the whole team when I say im really pleased and impressed with how quickly you have come up to speed on SEED and the work you've been contributing! I only have one note

File: seed/views/v3/organizations.py (hunk 1 above)
Comment (crutan) [reply seed/views/v3/organizations.py:865]: It sure doesn't.  Thanks for pointing that out.

File: seed/static/seed/js/controllers/inventory_reports_controller.js (hunk 2 above)
Comment (crutan) [reply seed/static/seed/js/controllers/inventory_reports_controller.js]: I've moved this into seed.js - I think that makes good sense.

File: seed/tests/test_organization_views.py [hunk 3]
```diff
@@ -283,4 +283,5 @@ def test_filtered_report(self):
 
         url += f"&filter_group_id={self.office_filter_group.id}"
         resp = self.client.get(url, content_type="application/json")
+        print(resp.json()["data"])
```
Comment (haneslinger) [inline seed/tests/test_organization_views.py:286]: remove

File: seed/tests/test_organization_views.py [hunk 4]
```diff
@@ -268,7 +268,7 @@ def setUp(self):
         self.office_filter_group = FilterGroup.objects.create(
             name="office",
             organization_id=self.org.id,
-            inventory_type=1,  # Property
+            inventory_type=0,  # Property
```
Comment (haneslinger) [inline seed/tests/test_organization_views.py:271]: 👍🏻

File: seed/models/filter_group.py [hunk 5]
```diff
@@ -21,12 +24,47 @@ class FilterGroup(models.Model):
...
-    def filtered_property_views(self, columns=[], views=None):
-        qd = QueryDict(mutable=True)
-        qd.update(self.query_dict)
-        filters, _annotations, _order_by = build_view_filters_and_sorts(qd, columns, "property")
-        filtered_views = views.filter(filters)
+    def views(self, views, columns=[]):
+        if VIEW_LIST_INVENTORY_TYPE[self.inventory_type][1] == "Property":
+            return self._filtered_property_views(views, columns)
+        elif VIEW_LIST_INVENTORY_TYPE[self.inventory_type][1] == "Tax Lot":
+            return self._filtered_taxlot_views(views, columns)
```
Comment (haneslinger) [inline seed/models/filter_group.py:31]: It's not strictly needed, just an fyi, if you want to only have one function, you could do something like [this](https://github.com/SEED-platform/seed/blob/Fix-update-display-name/seed/utils/match.py#L257), where the class is variable

**Review by haneslinger — APPROVED** [review]
Noice

File: seed/models/filter_group.py (hunk 5 above)
Comment (crutan) [reply seed/models/filter_group.py]: Oh, perfect, thank you for the pointer.  I'll implement that and collapse these down into a single function.

**Review by kflemin — APPROVED** [review]
🎉

**Pull request merged**